core/experiments/wp-cli.php

```diff
@@ -165,13 +196,15 @@ private function get_plural( $experiments ) {
 	 * @return bool true when all experiments exist, otherwise false
 	 */
 	private function check_experiments_exist( $experiments_manager, $experiments ) {
+        $allExperimentsExist = true;
```

Comment:
Indentation

Reply:
@louiswol94 Curious, this indent exists. Something seems off with how indents show Github vs when branch is pulled locally.

Reply:
![image](https://github.com/elementor/elementor/assets/130742176/06eee366-7a65-430b-b915-a2290ab0c045)